server/game/cards/11-TCaR/ChargingBear.js

```diff
@@ -0,0 +1,39 @@
+const DudeCard = require('../../dudecard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class ChargingBear extends DudeCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Charging Bear',
+            playType: ['shootout'],
+            condition: () => !this.booted,
+            targets: {
+                toAce: {
+                    activePromptTitle: 'Select token dude to ace',
+                    cardCondition: { location: 'play area', condition: card => card.isToken() },
+                    cardType: ['dude']
+                },
+                toBootHome: {
+                    activePromptTitle: 'Select an opposing dude to send home booted',
+                    cardCondition: { location: 'play area', controller: 'opponent', participating: true },
+                    cardType: ['dude']
+                }
+            },
+            message: context => {
+                this.game.addMessage('{0} uses {1} to ace {2}, send {3} home booted, and make {1} a stud!', context.player, this, context.targets.toAce, context.targets.toBootHome);
```

Comment:
```suggestion
                this.game.addMessage('{0} uses {1} to ace {2}, send {3} home booted, and make {1} a stud', context.player, this, context.targets.toAce, context.targets.toBootHome);
```
Making suggestion to restart the check and removing exclamation mark seems like a good candidate :)